Database/Corrections/sodQuestFixes.lua

```diff
@@ -3313,6 +3313,30 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
         [90217] = {
+            [questKeys.name] = "Shield Mastery",
+            [questKeys.startedBy] = {nil,{450011}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 5,
+            [questKeys.requiredRaces] = raceIDs.ORC + raceIDs.TROLL,
+            [questKeys.requiredClasses] = classIDs.SHAMAN,
+            [questKeys.objectivesText] = {"Find a Galvanic Icon, loot it and equip the Icon, follow its instructions to kill 10 enemies with Lightning Bolt Rank 1 to receive the rune."},
+            [questKeys.requiredSpell] = -410098,
+            [questKeys.zoneOrSort] = sortKeys.SHAMAN,
+        },
+        [90218] = {
+            [questKeys.name] = "Shield Mastery",
+            [questKeys.startedBy] = {nil,{450011}},
```

Comment:
I guess some merge failed.

```suggestion
            [questKeys.startedBy] = {nil,{450012}},
```